src/main.mjs

```diff
@@ -1,16 +1,30 @@
-// This is the example "Hello World" from:
-// https://expressjs.com/en/starter/hello-world.html
-const express = require('express');
-const morgan = require('morgan');
+//****************************************************************
+// INFO:
+// This file contains the main logic and routing for the server
+//****************************************************************
+
+//****************************************************************
+// Import and exports (ES6 style)
+//****************************************************************
+
+import express from 'express';
+import morgan from 'morgan';
+
+//****************************************************************
+// 
+//****************************************************************
 
 const logger = morgan('common');
 const app = express();
 
 app.use(logger); // Enable logging of requests.
 app.set('trust proxy', 'loopback'); // Trust locally hosted proxies for IP data.
 
+// The __dirname  = current/root directory
+const __dirname = import.meta.dirname;
+
 // Serve static files from the public directory. 
-// So that a get does not have to be create for the public directory.
+// So that a "get" does not have to be create for every file in the public directory.
```

Comment:
Dont quote get, maybe rephrase to GET to show its a html GET. Or call it instead a "get request".